chromium_src/chrome/browser/profiles/profile_window.cc

```diff
@@ -42,6 +44,10 @@ void OpenBrowserWindowForTorProfile(ProfileManager::CreateCallback callback,
   profiles::OpenBrowserWindowForProfile(
       callback, always_create, is_new_profile, unblock_extensions,
       profile->GetOffTheRecordProfile(), status);
+  tor::TorProfileService* service =
+    TorProfileServiceFactory::GetForProfile(profile);
```

Comment:
lint nit - 4 spaces for continuation of a line

Reply:
fixed